Development/nmos/sdp_utils.h

```diff
@@ -5,6 +5,7 @@
 #include <functional>
 #include <map>
 #include <stdexcept>
+#include <boost/lexical_cast.hpp>
```

Comment:
I think you should be able to use `utility::conversions::to_string_t` or `utility::conversions:: details::to_string_t`, instead?

Reply:
Thanks, `utility::conversions::details::to_string_t` will do the trick.